plugins/lookup/passwordstore.py

```diff
@@ -175,17 +181,29 @@ def parse_params(self, term):
                 else:
                     raise AnsibleError("{0} is not a correct value for length".format(self.paramvals['length']))
 
+            # Collect pass environment variables from the plugin's parameters.
+            self.env = {}
```

Comment:
You can also do
```suggestion
            self.env = os.environ.copy()
```
to make sure that everything else that's in the environment is passed on. Otherwise it might break existing use-cases where users set the environment manually.

Reply:
nice one.